build.gradle

```diff
@@ -4,19 +4,21 @@ allprojects {
 }
 
 buildscript {
-    ext.kotlin_version = '1.2.50'
-    ext.dokka_version = '0.9.16'
-    ext.detekt_version = '1.0.0.RC5'
+    ext.versions = [
+        kotlin: '1.2.61',
```

Comment:
Most people that use `kotlin` upgrade almost immediately, especially because `intellij` pushes you to do it. I would argue if you want to be conservative you should rather use `java`, but i'm happy to revert this if you feel strongly about it.